static/js/settings_config.ts

```diff
@@ -786,3 +786,34 @@ export const desktop_icon_count_display_values = {
         description: $t({defaultMessage: "None"}),
     },
 };
+
+export const system_user_groups_list = [
+    {
+        name: "@role:internet",
+        display_name: $t({defaultMessage: "Everyone on the internet"}),
+    },
+    {
+        name: "@role:everyone",
+        display_name: $t({defaultMessage: "Admins, moderators, members and guests"}),
+    },
+    {
+        name: "@role:members",
+        display_name: $t({defaultMessage: "Admins, moderators and members"}),
+    },
+    {
+        name: "@role:fullmembers",
+        display_name: $t({defaultMessage: "Admins, moderators and full members"}),
+    },
+    {
+        name: "@role:moderators",
+        display_name: $t({defaultMessage: "Admins and moderators"}),
+    },
+    {
+        name: "@role:administrators",
+        display_name: $t({defaultMessage: "Admins"}),
+    },
+    {
+        name: "@role:owners",
+        display_name: $t({defaultMessage: "Owners"}),
```

Comment:
These options are currently sorted as the `stream_post_policy` setting, with option which allows most roles at the top and with least role at the last. Any suggestions on the names, here like do we want `Admins only` instead of `Admins`. And also for `Admins, moderators, members and guests` optiov, we use `Everyone` for the `stream_post_policy` setting while for some org settings, we mention the four roles. Here we also have `Everyone on the internet option` (though not used in this setting currently).